pymeasure/instruments/instrument.py

```diff
@@ -61,11 +123,31 @@ class Instrument(object):
     :param adapter: A string, integer, or :py:class:`~pymeasure.adapters.Adapter` subclass object
     :param string name: The name of the instrument. Often the model designation by default.
     :param includeSCPI: A boolean, which toggles the inclusion of standard SCPI commands
-    :param \\**kwargs: In case ``adapter`` is a string or
-        integer, additional arguments passed on to
-        :py:class:`~pymeasure.adapters.VISAAdapter` (check there for details). Discarded otherwise.
+    :param \\**kwargs: In case ``adapter`` is a string or integer, additional arguments passed on
+        to :py:class:`~pymeasure.adapters.VISAAdapter` (check there for details).
+        Discarded otherwise.
     """
 
+    # Variable holding the list of DynamicProperty parameters that are configurable
+    # by users
```

Comment:
Still feels weird as a class attribute of Instrument because this is something related to properties. Maybe a better place would be as a class attribute of DynamicProperty? Then you would not need to pass it to that on construction, and could obtain it as DynamicProperty._fget_params_list (and other) in Instrument._compute_special_names.